webui/react/src/pages/DesignKit.tsx

```diff
@@ -2013,7 +2015,157 @@ const ModalSection: React.FC = () => {
   );
 };
 
+const LongLoadingComponent = () => {
+  const [loaded, setLoaded] = useState(false);
+  useEffect(() => {
+    let active = true;
+    setTimeout(() => {
+      if (active) setLoaded(true);
+    }, 5000);
+    return () => {
+      active = false;
+    };
+  });
+
+  return <div>This component is {loaded ? 'done loading!!!!!! wowza!!' : 'not loaded :('}</div>;
+};
+
+const AccordionSection: React.FC = () => {
+  const [controlStateSingle, setControlStateSingle] = useState(false);
+  const [controlStateGroup, setControlStateGroup] = useState(1);
+  return (
+    <ComponentSection id="Accordion" title="Accordion">
+      <AntDCard>
+        <p>
+          An <code>{'<Accordion>'}</code> hides content behind a header. Typically found in forms,
+          they hide complex content until the user interacts with the header.
+        </p>
+      </AntDCard>
+      <AntDCard title="Singular usage">
+        <p>
+          An <code>{'<Accordion>'}</code> requires a title and content to show:
+        </p>
+        <Accordion title="Title">Children</Accordion>
+        <p>
+          By default, <code>{'<Accordion>'}</code> components control their open state themselves,
+          but can be controlled externally:
+        </p>
+        <Checkbox
+          checked={controlStateSingle}
+          onChange={(e) => setControlStateSingle(e.target.checked)}>
+          Check me to open the accordion below!
+        </Checkbox>
+        <Accordion open={controlStateSingle} title="Controlled by the above checkbox">
+          Hello!
+        </Accordion>
+        <p>You can also render an uncontrolled accordion as open by default:</p>
+        <Accordion defaultOpen title="Open by default">
+          You should see me on page load.
+        </Accordion>
+        <p>
+          By default, the content of an <code>{'<Accordion>'}</code> isn&apost;t mounted until
```

Comment:
I believe you have an extra `t` here because this is rendering as "isn&apost;t" in the page.

Reply:
yep!
```suggestion
          By default, the content of an <code>{'<Accordion>'}</code> isn&apos;t mounted until
```